code/datums/components/drift.dm

```diff
@@ -57,7 +57,7 @@
 	SIGNAL_HANDLER
 	var/atom/movable/movable_parent = parent
 	movable_parent.inertia_moving = FALSE
-	UnregisterSignal(movable_parent, list(COMSIG_MOVABLE_MOVED, COMSIG_MOVABLE_NEWTONIAN_MOVE))
+	UnregisterSignal(movable_parent, (COMSIG_MOVABLE_MOVED))
```

Comment:
wrapped signal?

Reply:
Ah- didn't notice I have an extra set of parenthesis leftover; I'll fix that now